.gitignore

```diff
@@ -56,6 +56,7 @@ bazel-*
 /tilt.d
 tilt-settings.json
 tilt_config.json
+*.kubeconfig
```

Comment:
can you please move this up to line 32 with the other kubeconfig files?

Reply:
done